VRDR/DeathRecord_submissionProperties.cs

```diff
@@ -5404,43 +5401,197 @@ public string DeathLocationTypeHelper
+        /// <value>Set decedent's age at time of death in hours.</value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>ExampleDeathRecord.AgeAtDeathHours = 11;</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Age At Death: {ExampleDeathRecord.AgeAtDeathHours} hours");</para>
+        /// </example>
+        [Property("Age At Death Hours Helper", Property.Types.Int32, "Decedent Demographics", "Age At Death in Hours.", true, IGURL.DecedentAge, true, 2)]
+        [FHIRPath("Bundle.entry.resource.where($this is Observation).where(code.coding.code='39016-1')", "")]
+        public int AgeAtDeathHours
+        {
+            get {
+                if (AgeAtDeathObs?.Value != null)
+                {
+                    Quantity quantity = (Quantity)AgeAtDeathObs.Value;
+                    if (quantity.Code == "h")
+                    {
+                        return Convert.ToInt32(quantity.Value);
+                    }
+                }
+                return 0; // not sure how to best handle this case
+            }
+            set
+            {
+                AgeAtDeath = new Dictionary<string, string>() {
+                    { "value", Convert.ToString(value) },
+                    { "code", "h" }
+                };
+            }
+        }
+
+        /// <summary>Age At Death Minutes Helper</summary>
+        /// <value>Set decedent's age at time of death in minutes.</value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>ExampleDeathRecord.AgeAtDeathMinutes = 11;</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Age At Death: {ExampleDeathRecord.AgeAtDeathMinutes} minutes");</para>
+        /// </example>
+        [Property("Age At Death Minutes Helper", Property.Types.Int32, "Decedent Demographics", "Age At Death in Minutes.", true, IGURL.DecedentAge, true, 2)]
+        [FHIRPath("Bundle.entry.resource.where($this is Observation).where(code.coding.code='39016-1')", "")]
+        public int AgeAtDeathMinutes
+        {
+            get {
+                if (AgeAtDeathObs?.Value != null)
+                {
+                    Quantity quantity = (Quantity)AgeAtDeathObs.Value;
+                    if (quantity.Code == "min")
+                    {
+                        return Convert.ToInt32(quantity.Value);
+                    }
+                }
+                return 0; // not sure how to best handle this case
+            }
+            set
+            {
+                AgeAtDeath = new Dictionary<string, string>() {
+                    { "value", Convert.ToString(value) },
+                    { "code", "min" }
+                };
             }
         }
 
         /// <summary>Decedent's Age At Death Edit Flag.</summary>
-        /// <value>the decedent's age at death edit flag. A Dictionary representing a code, containing the following key/value pairs:
```

Comment:
The AgeAtDeathEditFlag code was actually correct, it's not an override, it's just a flag that can be set to 0 or 1, see http://build.fhir.org/ig/HL7/vrdr/StructureDefinition-vrdr-decedent-age.html